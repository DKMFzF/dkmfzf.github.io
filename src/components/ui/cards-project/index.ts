export { CardsProjectUI } from './cards-project';
