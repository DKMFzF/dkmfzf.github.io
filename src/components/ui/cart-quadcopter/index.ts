export { CartQuadcopter } from './cart-quadcopter';
