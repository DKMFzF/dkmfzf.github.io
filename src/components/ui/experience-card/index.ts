export { ExperienceCardUI } from './eperience-card';
