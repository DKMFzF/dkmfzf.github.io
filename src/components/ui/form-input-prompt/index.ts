export { FormInput } from './from-input';
