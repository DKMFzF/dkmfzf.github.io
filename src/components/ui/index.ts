export * from './preloader';
export * from './modal-overlay';
export * from './pages';
export * from './app-header';
export * from './modal';
export * from './icon';
export * from './link';
export * from './section-about-me';
