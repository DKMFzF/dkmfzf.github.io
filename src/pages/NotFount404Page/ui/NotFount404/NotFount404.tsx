import { FC } from 'react';
import { Helmet } from 'react-helmet-async';
import { META_SITE_DATA } from '@config';

export const NotFound404Page: FC = () => (
	<>
		<Helmet>
			<title>{META_SITE_DATA.titles.notFound}</title>
		</Helmet>
		<h3 className={`pb-6 text text_type_main-large`}>
			Страница не найдена. Ошибка 404.
		</h3>
	</>
);
