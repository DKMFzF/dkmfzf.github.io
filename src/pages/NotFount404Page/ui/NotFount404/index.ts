export { NotFound404Page } from './NotFount404';
