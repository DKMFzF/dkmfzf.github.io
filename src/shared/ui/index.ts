export * from './CommonPage';
export * from './Icon';
export * from './Preloader';
// export * from './Text';
