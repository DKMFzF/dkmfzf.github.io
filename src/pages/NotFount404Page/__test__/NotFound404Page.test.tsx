import { HelmetProvider } from 'react-helmet-async';
import { expect } from '@jest/globals';
import { NotFound404Page } from '../ui/NotFount404';
import { render } from '@testing-library/react';

describe('[NotFound404]', () => {
	it('Renders not-found-404 correctly and matches snapshot', () => {
		const { asFragment } = render(
			<HelmetProvider>
				<NotFound404Page />
			</HelmetProvider>
		);
		expect(asFragment()).toMatchSnapshot();
	});
});
