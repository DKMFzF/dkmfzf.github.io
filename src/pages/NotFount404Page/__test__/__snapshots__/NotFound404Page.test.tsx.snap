// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[NotFound404] Renders not-found-404 correctly and matches snapshot 1`] = `
<DocumentFragment>
  <h3
    class="pb-6 text text_type_main-large"
  >
    Страница не найдена. Ошибка 404.
  </h3>
</DocumentFragment>
`;
